target/linux/generic/files/drivers/net/phy/rtl8366_smi.c

```diff
@@ -1550,6 +1550,12 @@ int rtl8366_smi_probe_of(struct platform_device *pdev, struct rtl8366_smi *smi)
 		goto try_gpio;
 	}
 
+	if (of_property_read_u32(np, "phy_id", &smi->phy_id) < 0) {
+		smi->phy_id = MDC_REALTEK_PHY_ADDR;
+	}
```

Comment:
Please drop the `< 0` and the curly braces. of_property_read_u32 only returns negative values on failure or 0 on success.